libs/components/src/next/properties/vl-properties.model.ts

```diff
@@ -0,0 +1,11 @@
+export type Props = Column[];
+
+export interface Column {
+    class?: string; // column / column--full-width / collapsed
```

Comment:
Hier kan je ipv string het volgende meegeven:
`'column' | 'column--full-width' | 'collapsed'`

Reply:
ook combinaties kunnen: "column collapsed"
- het idee is dat die string 'as-is' wordt overgenomen in de gerenderde html
- eigenlijk 'groepeer' je properties waarbij er een 'div' rond gezet wordt die die class krijgt, je zou er ook voor kunnen opteren om een eigen css klasse te voorzien en die daar op te zetten - ik wou dus expliciet niet afdwingen dat het beperkt is tot 'onze' classes
- in een oudere versie noemde ik dat ook expiciet Group (ipv Column), dat gaf het concept beter weer, maar ik vond dat dan weer raar in code omdat de enige groepen die wij voorzien kolommen zijn